Refactoring

NatalyLo opened the pull request:
So far we have refactored everything related to discrepancy resolution and also saving to yaml after discrepancy resolution. All the interfaces reman

NatalyLo committed:
modified the structure

NatalyLo committed:
modified the structure

ppegolo committed:
Run pre-commit

Branch rebased to dev

ppegolo committed:
Run pre-commit

ppegolo committed:
Update docs and type hints for question.py

ppegolo committed:
Update docs and type hints for question.py

resolved question.py

ppegolo committed:
Update docs and type hints in publisher.py

ppegolo committed:
Update docs and type hints in publisher.py

ppegolo committed:
Update docs and type hints in journal.py

ppegolo committed:
Update docs and type hints in journal.py

ppegolo committed:
Update docs and type hints in csv_handler.py

ppegolo committed:
Update docs and type hints in yaml_handler.py

ppegolo committed:
Update docs and type hints in grouper.py

r
rebased grouper

ppegolo committed:
Update docs and type hints in grouper.py

ppegolo committed:
Fix typo

ppegolo committed:
Add question_id to Question class

rebased question.py

ppegolo committed:
Add question_id to Question class

ppegolo committed:
Minor aesthetic changes

ppegolo committed:
Minor aesthetic changes

ppegolo committed:
Move metadata outside of the package

ppegolo committed:
Move metadata outside of the package

ppegolo committed:
Update question types (use question id)

ppegolo committed:
Update question types (use question id)

NatalyLo committed:
updated yaml saver

NatalyLo committed:
updated yaml saver

NatalyLo committed:
removed files

NatalyLo committed:
added tests

NatalyLo committed:
added tests

NatalyLo committed:
added more tests and fixed grouper

NatalyLo committed:
added more tests and fixed grouper

NatalyLo committed:
added final example notebook

NatalyLo committed:
added final example notebook

NatalyLo committed:
hid functions import

NatalyLo committed:
hid functions import

NatalyLo committed:
linting

NatalyLo committed:
linting

ppegolo commented on src/daemon_analysis_tools/facade_functions.py:
Can't this be in `__init__.py`?

NatalyLo replied:
it seems it's possible, updated

NatalyLo committed:
moved all facade functions to init

NatalyLo committed:
moved all facade functions to init

NatalyLo committed:
fixed bugs and added end-to-end test for build_journal_dict

NatalyLo committed:
fixed bugs and added end-to-end test for build_journal_dict

NatalyLo committed:
linting

NatalyLo committed:
linting

lukas-hoermann committed:
Consistency check (#36)

* Added bentham encoding

* added yaml files

* updated fix_bentham notebook

* Change format of correct answer key in yaml files

* Updated a lot of code file

* Update patch_question_identifiers

* Updated all yaml files

* Fixed missing bentham encodings; added check_coverager.ipynb

* Moved fix notebooks in separate directory

* Added additional information to asserts in load_yaml

* Manually updated yaml file with missing information

* Created yaml files for Wiley, which I missed before

* updated check_inconsistencies

* updated check coverage

* Fixed Nature

* updated fix_inconsistencies

* Update rdp.csv, added additional naming conventions to data_processing

* modified the structure

* Run pre-commit

* Update docs and type hints for question.py

* Update docs and type hints in publisher.py

* Update docs and type hints in journal.py

* Update docs and type hints in csv_handler.py

* Update docs and type hints in yaml_handler.py

* Update docs and type hints in grouper.py

* Fix typo

* Add question_id to Question class

* ACS consistency checks

manually fixed data in wrong columns in rdp.csv;
added ACS yaml files;
added work flow to create ACS yaml files

* removed build folder

* add more journals to normalize journal function, add basic unit-test for data processing normalize_journal

* fix pre-commit again

* add README for dimensions data

* Minor aesthetic changes

* Move metadata outside of the package

* Update question types (use question id)

* update to analysis_example notebook

* added ieee yaml file

* updated rdp.csv

* updated rdp.csv

* Updated rdp.csv and added rdp.xlsx

Since some of our descriptions in the rdp.csv file have commas, the csv format may mess this up. Ich changed the the format so that there are quotes delimiting text. For safety, I also added a regular excel file.

* fixed Pleiades publishing

* added yaml files for MDPI

* some linting in normalize

* linting

* linting

* linting

* more linting

* added analysis workflows

* fix linting

* fix normalize.. to _normalize.. function names and data_processing_test

* fix union type operator for older python versions

* add missing import from typing import Optional

* formatting of notebooks and plots

---------

Co-authored-by: ppegolo <ppegolo@sissa.it>
Co-authored-by: NatalyLo <lopanitsyna.nataliya@gmail.com>
Co-authored-by: Markus Stricker <markus.stricker@rub.de>
Co-authored-by: JonathanSchmidt1 <schmidt.jonathan138@googlemail.com>

NatalyLo committed:
removed artifacts

NatalyLo committed:
minor conflicts resolved

NatalyLo committed:
moved test

NatalyLo committed:
linting

NatalyLo committed:
fixed representation
